target/linux/qualcommax/patches-6.6/0066-v6.13-arm64-dts-qcom-ipq-change-labels-to-lower-case.patch

```diff
@@ -0,0 +1,381 @@
+From 6f8c1ed25809181c187a59b1caaa1521756924bf Mon Sep 17 00:00:00 2001
+From: Krzysztof Kozlowski <krzysztof.kozlowski@linaro.org>
+Date: Tue, 22 Oct 2024 17:47:26 +0200
+Subject: [PATCH] arm64: dts: qcom: ipq: change labels to lower-case
```

Comment:
In 34d9172655af4e71f9b06229cbd5a716d359ac23 (#17912?), a patch that this one depends on is called `target/linux/qualcommax/patches-6.6/0077-v6.8-arm64-dts-qcom-ipq5018-enable-the-CPUFreq-support.patch`. I had to rename this file from `0066-*.patch` to `0078-*.patch` in order to have the patches applied in the correct order.